### PR TITLE
NXBT-1936: fix Content-Type header parsing

### PR DESCRIPTION
**Pull request by troger**

### CODE REVIEW

**Comment by efge** [comment]
👍

File: crowdin_updater.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def upload(self, inputfile, parentdir=None):
     # 1 if success
     # 0 if unknown status
     def parseXMLResponse(self, r):
-        if r.headers['content-type'] != 'text/xml':
+        if r.headers['content-type'].startswith('text/xml'):
```
Comment (mincong-h) [inline crowdin_updater.py:101]: Would it be better to use "contains" instead of "starts with"?

```py
if 'text/html' in r.headers['content-type']:
```

Comment (efge) [reply crowdin_updater.py:101]: No need, a Content-Type header will always starts with the media type (RFC 2045 §5.1)

Comment (troger) [reply crowdin_updater.py:101]: According to the spec, the `Content-Type` should start with `type/subtype`, so let's check that.

**Review by mincong-h — APPROVED** [review]

**Comment by atchertchian** [comment]
:+1:

**Review by atchertchian — APPROVED** [review]

**Pull request merged**